### DISCUSSION TITLE
High Container CPU Usage Normal?

### DISCUSSION THREAD

**Discussion by imaohw1337** [Security-Onion-Solutions/securityonion#8786] [Unsupported Versions]
Why does the so-elasticsearch container CPU usage on my search node regularly go from 500% all the way up to 3000%?
Same with so-logstash container CPU usage jumping up to 1000%? Is this normal? If not please advise how one can give more cores to a container. My overall CPU usage on my search node according to Grafana is around 20% so it looks like I have CPU to spare.

**Comment by dougburks** [comment]
Containers should automatically take advantage of any available cores.

How many search nodes do you have?

What is the log volume they are consuming?

Have you made any changes to the default configuration?

Have you checked logs for additional clues?
